cluster/gce/gci/configure-helper.sh

```diff
@@ -2399,6 +2399,22 @@ EOF
   fi
   if [[ "${ENABLE_METRICS_SERVER:-}" == "true" ]]; then
     setup-addon-manifests "addons" "metrics-server"
+    base_metrics_server_cpu="40m"
+    base_metrics_server_memory="40Mi"
+    metrics_server_memory_per_node="4"
+    metrics_server_min_cluster_size="16"
+    if [[ "${ENABLE_SYSTEM_ADDON_RESOURCE_OPTIMIZATIONS:-}" == "true" ]]; then
+      # TODO: Set each of the following variables to optimized values.
+      base_metrics_server_cpu="40m"
+      base_metrics_server_memory="40Mi"
```

Comment:
Same as above

Reply:
Can you please clarify which previous comment you're referring to?